docs/templates/plugins.rst.j2

```diff
@@ -30,8 +30,10 @@
             {% else %}
     Default: {% for default_item in actual_default %}``{{ default_item.pattern | default(default_item) }}``{% if not loop.last %}, {% endif %}{% endfor %}
             {% endif %}
+        {% elif is_enum(actual_default) %}
+    Default: ``{{ actual_default.value }}``
```

Comment:
I've tried to add 
```
    {% elif is_enum(metadata.type) %}
{{ option }}: ``{{ metadata.type.__members__.keys() | join('|') }}``
```
Not sure what's up.

Btw, I notice a typo below, `suboptional` -> `suboptimal` @happz ?

Reply:
Oh yeah, that looks like a typo.

The template addition looks good to me, it didn't work? `member.value for member in ResultOutcome.__members__.values()` would be probably better, in the `PASS = 'pass'` we want the `'pass'` string, which is the `value` of a member.

Reply:
@happz I thought it would be good to use `choices` here, but wasn't sure how to make it scalable for all enums:  https://github.com/teemtee/tmt/pull/3239/commits/f197ee035b395a74e60f60195bc1d26714931b0d

Reply:
Works nice, thanks!